src/libstd/os/linux/raw.rs

```diff
@@ -205,7 +205,8 @@ mod arch {
     }
 }
 
-#[cfg(target_arch = "x86_64")]
+#[cfg(any(target_arch = "x86_64", target_arch = "powerpc64",
+          target_arch = "powerpc64le"))]
```

Comment:
Yes, just to confirm:
blkcnt_t 64
blksize_t 64
ino_t 64
nlink_t 64
off_t 64
time_t 64